src/main/scala/org/pantsbuild/jarjar/ScalaSigProcessor.scala

```diff
@@ -0,0 +1,19 @@
+package org.pantsbuild.jarjar
+
+import org.objectweb.asm.{ClassReader, ClassWriter}
+import org.pantsbuild.jarjar.util.{EntryStruct, JarProcessor}
+
+class ScalaSigProcessor(rules: Seq[Rule]) extends JarProcessor {
+  override def process(struct: EntryStruct): Boolean = {
+    if (!struct.name.endsWith(".class") || struct.skipTransform) true
```

Comment:
Maybe allow `*.sig` files here? - https://github.com/scala/scala/pull/7712

Reply:
I'll add it to the list. I have some reading up to do here.